tapdb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -107,14 +107,24 @@ CREATE TABLE IF NOT EXISTS asset_groups (
 CREATE TABLE IF NOT EXISTS asset_group_sigs (
     sig_id INTEGER PRIMARY KEY,
 
-    genesis_sig BLOB NOT NULL, 
+    witness_id INTEGER NOT NULL REFERENCES asset_group_witnesses(witness_id),
 
     -- TODO(roasbeef): not needed since already in assets row?
     gen_asset_id INTEGER NOT NULL REFERENCES genesis_assets(gen_asset_id) UNIQUE,
 
     group_key_id INTEGER NOT NULL REFERENCES asset_groups(group_id)
 );
 
+CREATE TABLE IF NOT EXISTS asset_group_witnesses (
+    witness_id INTEGER PRIMARY KEY,
+
+    tapscript_root BLOB,
```

Comment:
Do we hope that we can re-use some of these witnesses? Otherwise the alternative would be to just add both of these fields to the `asset_group_sigs` table and reduce the number of tables needed.

Reply:
I don't think so - I think actually the `tapscript_root` should go into `asset_groups` since it is static per group key, and then `witness_stack` could indeed be a field in `asset_group_sigs`.

One consideration is that the witnesses aren't really bounded size right now, and I suppose if they are in a separate table then we have more options in not fetching them for some queries? Not sure if that's a relevant concern.

Reply:
Agree that it would make sense to have the static `tapscript_root` with the asset group itself. No strong opinion on where the witness stack goes.